package.json

```diff
@@ -59,7 +59,7 @@
     "react-redux-firebase": "^3.10.0",
     "react-resize-detector": "^5.0.7",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.4.1",
+    "react-scripts": "4.0.3",
```

Comment:
[note] This was the main intended change in this PR, the rest is related to changed dependencies, lint rules, etc because of this.